doc/howto/network_bridge_firewalld.md

```diff
@@ -1,52 +1,109 @@
-(network-bridge-firewalld)=
-# How to configure Firewalld
+---
+discourse: 10034,9953
+---
 
-## Allow DHCP, DNS with Firewalld
+(network-bridge-firewall)=
+# How to configure your firewall
 
-In order to allow instances to access the DHCP and DNS server that LXD runs on the host when using firewalld
-you need to add the host's bridge interface to the `trusted` zone in firewalld.
+If you run a firewall on your system, you must configure it to allow network traffic from and to the LXD bridge.
 
-To do this permanently (so that it persists after a reboot) run the following command:
+If you are using LXD's firewall in addition (that is, if you have configured the `ipv4.firewall` or `ipv6.firewall` {ref}`configuration options <network-bridge-options>`), you must ensure that there are no conflicts between LXD's firewall rules and the existing ones.
 
-```
-firewall-cmd --zone=trusted --change-interface=<LXD network name> --permanent
-```
+See the following sections for instructions on how to achieve this for firewalld and UFW, respectively.
 
-E.g. for a bridged network called `lxdbr0` run the command:
+## Allow traffic from/to the LXD bridge
 
-```
-firewall-cmd --zone=trusted --change-interface=lxdbr0 --permanent
-```
+To allow instances to access the DHCP and DNS server that LXD runs on the host, you must configure your firewall to allow network traffic from and to the LXD bridge.
+This will then allow LXD's own firewall rules to take effect for all instances that use the network bridge.
```

Comment:
Is this correct for UFW as well? Or in other words, does the UFW section below actually make sense? ;)